### PR TITLE
[DO NOT MERGE YET] Moving to gitbook

### PR DESCRIPTION
**Pull request by slackmoehrle**
- added reading language choosing
- added needed items for programming language choosing to be better
- Modify first chapter of PG to show this.

### CODE REVIEW

File: en/SUMMARY.md [hunk 1]
```diff
@@ -1,96 +1,113 @@
...
+- [Android - command-line]()
+- [Android - Visual Studio]()
+- [Android - Studio]()
+- [Android - Eclipse]()
+- [IOS]()
+- [Mac OS X]()
+- [Linux]()
+- [Tizen]()
+- [Windows]()
+- [Windows Phone]()
```
Comment (minggo) [inline en/SUMMARY.md]: Tizen and Windows Phone is not supported now, so i don't think they are needed.

Comment (slackmoehrle) [reply en/SUMMARY.md]: ok, removed

File: en/SUMMARY.md [hunk 2]
```diff
@@ -1,96 +1,113 @@
...
 - [Sprite3D]()
 - [Animation]()
-- [...]()
+- [Camera]()
+- [Cubemap Texture]()
+- [Skybox]()
+- [Lighting]()
+- [Terrain]()
+- [Tools for creating 3D Sprites]()
+- [Advanced 3D Topics]()
```
Comment (minggo) [inline en/SUMMARY.md:72]: They are existing content, right? We will not add new content for 3D.

Comment (slackmoehrle) [reply en/SUMMARY.md:72]: yes already existing content

Comment (slackmoehrle) [reply en/SUMMARY.md:72]: The only thing I will add to 3D is a few more code example for things like touch events and perhaps a more complete camera example.

Comment (minggo) [reply en/SUMMARY.md:72]: got it

File: en/SUMMARY.md [hunk 3]
```diff
@@ -1,96 +1,113 @@
...
- - [Buttons]()
- - [...]()
+- [Widgets, oh, my!](./ui_components.md)
+- [Label]()
+- [Menu]()
+- [Buttons]()
+- [Checkbox]()
+- [Loading Bar]()
+- [Slider]()
+- [TextField]()
```
Comment (minggo) [inline en/SUMMARY.md:44]: How about other UI widgets, such as `ListView`, `PageView` and so on.

**Comment by minggo** [comment]
Is there any doc about how to do with multiple resolution adaption?

**Comment by slackmoehrle** [comment]
There are some old docs in the wiki. But yes we really need this. I can write it.

________________________________

**Pull request merged**